pkgs/applications/misc/far2l/default.nix

```diff
@@ -73,7 +70,7 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     description = "Linux port of FAR Manager v2, a program for managing files and archives in Windows operating systems";
     homepage = "https://github.com/elfmz/far2l";
-    license = licenses.gpl2Plus; # NOTE: might change in far2l repo soon, check next time
+    license = licenses.gpl2Only; # NOTE: might change in far2l repo soon, check next time
```

Comment:
Can the comment be dropped?

Reply:
Seems OK according to elfmz/far2l#692; dropped